backend/app/rest/api/rest.go

```diff
@@ -308,6 +308,9 @@ func (s *Rest) routes() chi.Router {
 			rauth.Post("/comment", s.privRest.createCommentCtrl)
 			rauth.With(rejectAnonUser).Put("/vote/{id}", s.privRest.voteCtrl)
 			rauth.With(rejectAnonUser).Post("/deleteme", s.privRest.deleteMeCtrl)
+			rauth.With(rejectAnonUser).Get("/email", s.privRest.sendConfirmation)
```

Comment:
GET and POST symanticks doesn't make it clear. Probably both POST and change urls, i.e. /email/confirm and /email/subscribe will be less confusing